### ISSUE TITLE
How to use SimpleCLM

### ISSUE THREAD

**Issue by sinfone** [TadasBaltrusaitis/CLM-framework#38]

**Comment by TadasBaltrusaitis** [comment]
Use OpenFace instead, instructions are provided in the wiki - https://github.com/TadasBaltrusaitis/OpenFace/wiki

**Comment by simonmssu** [comment]
Hello,

I may have seen in action where the face with a blue box (just released
that is OpenFace). Just like to confirm if OpenFace also has the data of
the position and the direction of where the face is looking?

thanks
-simon

On Tue, May 31, 2016 at 11:43 AM, Tadas Baltrusaitis <
notifications@github.com> wrote:

> Use OpenFace instead, instructions are provided in the wiki -
> https://github.com/TadasBaltrusaitis/OpenFace/wiki
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/TadasBaltrusaitis/CLM-framework/issues/38#issuecomment-222729963,
> or mute the thread
> https://github.com/notifications/unsubscribe/ABqXE3Wv581xsHQ8qyC2O6TV9EXrSDiMks5qHFcygaJpZM4IqMtY
> .

## 

-simon

**Comment by TadasBaltrusaitis** [comment]
Yes it does, it has all the functionality of CLM-framework plus more

**Comment by simonmssu** [comment]
Nice. Thanks for the info.

thanks
-simon

On Tue, May 31, 2016 at 11:59 AM, Tadas Baltrusaitis <
notifications@github.com> wrote:

> Yes it does, it has all the functionality of CLM-framework plus more
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> https://github.com/TadasBaltrusaitis/CLM-framework/issues/38#issuecomment-222735093,
> or mute the thread
> https://github.com/notifications/unsubscribe/ABqXEweOUZwvHAccGIF--Tj7Ck0csUjXks5qHFr7gaJpZM4IqMtY
> .

## 

-simon
